tests/pfcwd/test_pfcwd_timer_accuracy.py

```diff
@@ -0,0 +1,208 @@
+import logging
+import pytest
+import time
+
+from common.fixtures.conn_graph_facts import fanout_graph_facts
+from common.helpers.assertions import pytest_assert
+from common.helpers.pfc_storm import PFCStorm
+from files.pfcwd_helper import start_wd_on_ports
+
+pytestmark = [pytest.mark.disable_loganalyzer]
+
+logger = logging.getLogger(__name__)
+
+@pytest.fixture(scope='module', autouse=True)
+def stop_pfcwd(duthost):
+    """
+    Fixture that stops PFC Watchdog before each test run
+
+    Args:
+        duthost (AnsibleHost): DUT instance
+    """
+    logger.info("--- Stop Pfcwd --")
+    duthost.command("pfcwd stop")
+
+@pytest.fixture(scope='class', autouse=True)
+def pfcwd_timer_setup_restore(setup_pfc_test, fanout_graph_facts, duthost, fanouthosts):
+    """
+    Fixture that inits the test vars, start PFCwd on ports and cleans up after the test run
+
+    Args:
+        setup_pfc_test (fixture): module scoped, autouse PFC fixture
+        fanout_graph_facts (fixture): fanout graph info
+        duthost (AnsibleHost): DUT instance
+        fanouthosts (AnsibleHost): fanout instance
+
+    Yields:
+        timers (dict): pfcwd timer values
+        storm_handle (PFCStorm): class PFCStorm instance
+    """
+    logger.info("--- Pfcwd timer test setup ---")
+    setup_info = setup_pfc_test
+    test_ports = setup_info['test_ports']
+    timers = setup_info['pfc_timers']
+    eth0_ip = setup_info['eth0_ip']
+    pfc_wd_test_port = test_ports.keys()[0]
+    neighbors = setup_info['neighbors']
+    fanout_info = fanout_graph_facts
+    dut = duthost
+    fanout = fanouthosts
+    peer_params = populate_peer_info(neighbors, fanout_info, pfc_wd_test_port)
+    storm_handle = set_storm_params(dut, fanout_info, fanout, peer_params)
+    timers['pfc_wd_restore_time'] = 400
+    start_wd_on_ports(dut, pfc_wd_test_port, timers['pfc_wd_restore_time'],
+                      timers['pfc_wd_detect_time'])
+    # enable routing from mgmt interface to localhost
+    dut.shell("sysctl -w net.ipv4.conf.eth0.route_localnet=1")
```

Comment:
Ansible has sysctl [module](https://docs.ansible.com/ansible/2.8/modules/sysctl_module.html). Can it be used here?